odoo/fields.py

```diff
@@ -1118,6 +1098,29 @@ def __set__(self, records, value):
     # Computation of field values
     #
 
+    def recompute(self, records):
+        """ Process the pending computations of ``self`` on ``records``. This
+        should be called only if ``self`` is computed and stored.
```

Comment:
shouldn't we explicitly check for this at the beginning of the function's body then? then you can spare the ifs in `mapped` and `__get__`, if the field isn't computed/stored it'd be a `noop`

Reply:
Good point, but I have two reasons to do it this way:
* this test is done anyway in `BaseModel.recompute`;
* on the fields that do not fit, this saves a method call.

I believe that the second point makes a difference on a majority of fields.  And this counts, because the field getter is a very common operation on records.